docs/drafts/upgrade-support.md

```diff
@@ -0,0 +1,67 @@
+# Upgrade support
+
+This document explains how OLM 1.0 handles upgrades.
+
+OLM 1.0 introduces a simplified UX for package authors and package admins to implicitly define upgrade edges via [Semantic Versioning](https://semver.org/).
+
+It also introduces an API to enable independently verified upgrades and downgrades.
+
+## Upgrade constraint semantics
+
+As of Operator Controller release 0.8.0, OLM 1.0 supports the following upgrade constraint semantics:
+
+* [Semantic Versioning](https://semver.org/) (Semver)
+* The `replaces` directive from the [legacy OLM 0 semantics](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/#methods-for-specifying-updates)
+
+The Kubernetes manifests in this repo enable Semver support by default. Cluster admins can control which semantics to use by passing one of the following arguments to the `manager` binary:
```

Comment:
This might be out of scope for this issue, but it might be worth explaining in a follow up issue why a cluster would want to deviate from the upgrade constraint semantics specified by the package author.

Reply:
I added the following:

> We plan to remove the `ForceSemverUpgradeConstraints` feature gate and enable package authors to specify which upgrade constraint semantics they wish to use on the catalog level.